src/components/bug-report/index.tsx

```diff
@@ -0,0 +1,22 @@
+import { Button, ButtonOrder, ButtonSize } from '../button';
+import * as React from 'react';
+import { getSpace, SpaceSize } from '../space';
+import styled from 'styled-components';
+
+export interface BugReportProps {
+	onClick?: React.MouseEventHandler<HTMLElement>;
+	title: string;
+}
+
+const StyledBugReport = styled.div`
+	justify-self: right;
+	margin-right: -${getSpace(SpaceSize.XXL) * 3 - getSpace(SpaceSize.L)}px; // align to top right corner
```

Comment:
The padding should be removed in the `chrome` component

Reply:
Yes, but the padding leads to the fact, that the "< Pages" indicator is under the 3 mac close icons. Or if we only set the right padding differently, the center title is not properly aligned (a bit too right). That’s why I chose that solution.